src/_app.scss

```diff
@@ -4,4 +4,13 @@
   display: block;
   overflow: hidden;
   min-height: 100%;
+
+  main {
```

Comment:
That's a great add. I thought also that the .container class is quite bad way to do the main container styling.